source/_drafts/Introducing-Video-js-HTTP-Streaming-VHS.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Introducing Video.js HTTP Streaming (VHS)
+author:
+  name: Garrett Singer
+  github: gesinger
+tags:
+  - vhs
+  - videojs-http-streaming
+  - http-streaming
+  - playback
+---
+
+"How do I get my video to play with Video.js?"
+
+This is one of the most frequent questions we get when working on Video.js. And it's a good question.
+
+If someone checks out a copy of Video.js, their content may play on one browser and not another. To get cross browser support for a specific type of content, they must stumble upon a relevant source handler (for instance, [videojs-contrib-hls](https://github.com/videojs/videojs-contrib-hls) for HLS content). Only after including the source handler will their video play across all major browsers.
+
+We want Video.js to be easy to use, and that isn't the easiest setup. So we decided that it's time to address the issue.
+
+This was the motivation behind integrating [Video.js HTTP Streaming](https://github.com/videojs/http-streaming) (nicknamed VHS) inside of Video.js 7 by default.
+
+## What is VHS?
+
+VHS is a source handler forked from the [videojs-contrib-hls](https://github.com/videojs/videojs-contrib-hls) repository. While videojs-contrib-hls was originally designed to add HLS playback on all browsers, we realized that the engine could also play other formats just as well.
```

Comment:
I would change the first sentence to "VHS is the successor to videojs-contrib-hls".